### DISCUSSION TITLE
how to keep custom nova components up to date with nova updates

### DISCUSSION THREAD

**Discussion by AbdallahAli912** [laravel/nova-issues#5539] [Q&A] [answered]
- Laravel Version: 9
- Nova Version: 4
- PHP Version: 8..2

### Description:

We used a way to  override some nova components.
Is there a way for accessing nova GitHub  repo
to keep track of changes made to that components  by nova team ?
 or Is there a detailed changes report  that can help us to keep  our custom components up to date with nova updates ?

**Comment by crynobone** [comment, marked as answer]
Nova component is not under public API and isn't documented. If you need to override any component, you should always ensure that dusk/cypress tests or vue tests (jest etc) cover those usages to ensure it continues to work after each update.
